wp-content/plugins/core/src/Theme/Resources/Colors.php

```diff
@@ -33,4 +33,44 @@ public static function get_color_options_subset( array $colors = [] ) {
 		return array_intersect_key( self::get_color_options(), array_flip( $colors ) );
 	}
 
+	/**
+	 * Create an array of color options for use with the ACF Swatch field.
+	 *
+	 * @param array $colors
+	 *
+	 * @return array
+	 */
+	public static function get_color_options_for_acf( array $colors = [] ) {
```

Comment:
This is great! Is there any way we could change the method names so that they don't necessarily mention ACF? These methods seem like they could be useful for a whole host of things, so it'd be nice to generify them a bit.

Reply:
Good thought. I've tweaked the function names accordingly.